fenix/app/src/androidTest/java/org/mozilla/fenix/extensions/ExtensionProcessTest.kt

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.extensions
+
+import android.content.Context
+import mozilla.components.concept.engine.EngineSession
+import org.json.JSONObject
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.mozilla.experiments.nimbus.HardcodedNimbusFeatures
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.gecko.GeckoProvider
+import org.mozilla.fenix.helpers.TestHelper
+import org.mozilla.fenix.nimbus.FxNimbus
+
+/**
+ * Instrumentation test for testing the extension process can be controlled with Nimbus.
```

Comment:
```suggestion
 * Instrumentation test for verifying that the extension process can be controlled with Nimbus.
```

Reply:
Also extra nit: I (and likely others) have been using "extensions process" in many places already (instead of "extension process")

Reply:
Thanks for catching this 👍🏽